test/passes/inline_return_call_ref.bin.txt

```diff
@@ -0,0 +1,15 @@
+(module
```

Comment:
Why is there no .wast file for this test?

Reply:
it's a wasm binary file atm - but yeah, it could be a wat, let me fix that...